src/main/java/racing/domain/TestGameRule.java

```diff
@@ -0,0 +1,19 @@
+package racing.domain;
+
+import java.util.Random;
+
+public class TestGameRule implements GameRule{
```

Comment:
```suggestion
public class TestGameRule implements GameRule {
```
- 커밋하기 전에 포맷 맞추는 습관을 들이셔요~ 아니면, 린트를 적용해두는 것도 좋습니다.